packages/bsb-core/bsb/config/_attrs.py

```diff
@@ -986,24 +1016,104 @@ def tree(self, instance):
         return val
 
 
+class cfgreflist(cfglist):
```

Comment:
I see that you've overwritten most methods, and that they for example don't use `_preset` anymore? Could you explain a bit why you did it this way? I had sort of hoped that we'd only have to modify `_preset` or `_postset` lightly.